csharp/Test/Integration/Examples/CoreExamplesTest.cs

```diff
@@ -0,0 +1,304 @@
+                Assert.Fail("You can handle exceptions here. However, we do not expect exceptions in CI, so we fail.");
+            }
+        }
+        
+        [Test]
+        public void ManualExample()
+        {
+            string dbName = "access-management-db";
+            string serverAddr = "127.0.0.1:1729";
+
+            try
+            {
+                ITypeDBDriver driver = Drivers.CoreDriver(serverAddr);
+                driver.Databases.Create(dbName);
+
+                IDatabase database = driver.Databases.Get(dbName);
+
+                ITypeDBSession schemaSession = driver.Session(dbName, SessionType.Schema);
+                ITypeDBTransaction schemaWriteTransaction = schemaSession.Transaction(TransactionType.Write);
+
+                schemaWriteTransaction.Query.Define("define person sub entity;").Resolve();
+
+                string longQuery = "define name sub attribute, value string; person owns name;";
+                schemaWriteTransaction.Query.Define(longQuery).Resolve();
+
+                schemaWriteTransaction.Commit(); // No need to close manually if committed
+                schemaSession.Close();
+
+                ITypeDBSession dataSession = driver.Session(dbName, SessionType.Data);
+                ITypeDBTransaction dataWriteTransaction = dataSession.Transaction(TransactionType.Write);
+
+                string query = "insert $p isa person, has name 'Alice';";
+                IConceptMap[] insertResults = dataWriteTransaction.Query.Insert(query).ToArray();
+                ProcessPersonInsertResult(insertResults, "p", "person", "Alice");
+
+                dataWriteTransaction.Commit();
+                dataSession.Close();
+
+                dataSession = driver.Session(dbName, SessionType.Data);
+                dataWriteTransaction = dataSession.Transaction(TransactionType.Write);
+
+                insertResults = dataWriteTransaction.Query.Insert("insert $p isa person, has name 'Bob';").ToArray();
+                ProcessPersonInsertResult(insertResults, "p", "person", "Bob");
+
+    //            dataWriteTransaction.Commit(); // Not committed
+
+                ITypeDBTransaction readTransaction = dataSession.Transaction(TransactionType.Read);
+                IConceptMap[] matchResults =
+                    readTransaction.Query.Get("match $p isa person, has name $n; get $n;").ToArray();
+
+                ProcessPersonMatchResult(matchResults, "n", "name", "Alice");
+
+                readTransaction.Close();
+                dataSession.Close();
+
+                database.Delete();
+                driver.Close();
+            }
+            catch (TypeDBDriverException e)
+            {
+                Console.WriteLine($"Caught TypeDB Driver Exception: {e}");
+                // ...
+                Assert.Fail("You can handle exceptions here. However, we do not expect exceptions in CI, so we fail.");
+            }
+        }
+
+        [Test]
+        public void DocExample()
```

Comment:
This test is for copypasting into README. The upper ones show the same workflow with different ways of doing things.